CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.0.2]
+
+### Changed
+
+- Upgraded Rust version to 1.64. This enables us to keep supporting v1.1.
```

Comment:
We did not upgrade to 1.64: https://github.com/firecracker-microvm/firecracker/blob/firecracker-v1.0/tools/devctr/Dockerfile.x86_64
We forked linux-loader as a workaround.

Reply:
I will drop that commit then.